templates/mesh-gateway-deployment.yaml

```diff
@@ -225,10 +225,10 @@ spec:
             {{- end }}
           resources:
             requests:
-              memory: "25Mi"
+              memory: "50Mi"
               cpu: "50m"
             limits:
-              memory: "25Mi"
+              memory: "50Mi"
```

Comment:
These limits were already increased for other gateways, and I've seen it fail due to OOM issues on openshift